### PR TITLE
add list of figures in readme - fixes #17

### LINKED ISSUE #17: Add list of figures in the readme
I'm not 100% sure, so didn't do a PR yet, because it might be too much.

It would be nice to have an overview of all the figures in the readme. Now you always have to dig into the code in order to see which symbols are supported.

Something like this might do

| Name  | Real OSes | Windows |
|-------|:---------:|:-------:|
| tick  |     ✔︎     |    √    |
| cross |     ✖     |    ×    |
| star |     ★     |    *    |
| ... |          |        |

### PR DESCRIPTION
**Pull request by SamVerschueren**
Added a `makefile.js` script that retrieves the figures from the module and adds them to the readme. The script also executes on `prepublish` to make sure it is executed before released to `npm`.

If you rather give it another name, suggestions are welcome :).

### CODE REVIEW

**Comment by SamVerschueren** [comment]
Tests are failing because somehow `npm run make` was fired. Probably due to the `prepublish` script. How should I fix this? Just un-ES2015ify the `makefile` or remove the `prepublish` script?

**Comment by sindresorhus** [comment]
Just remove running it on prepublish.

**Comment by SamVerschueren** [comment]
Done

**Comment by sindresorhus** [comment]
Seems to be failing on AppVeyor.

**Comment by SamVerschueren** [comment]
It's because I started this PR from `master` and thus it doesn't have an `appveyor.yml` file. So if any of these PRs get merged, I will rebase.

**Comment by sindresorhus** [comment]
Can you rebase?

**Comment by SamVerschueren** [comment]
Done.

28h309z

**Comment by SamVerschueren** [comment]
So true... It's still queued, wtf!?

**Comment by SamVerschueren** [comment]
One hour later and it's still running :D.

**Comment by kevva** [comment]
Woho, passed!

File: makefile.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+];
+
+Object.keys(darwin).forEach(key => {
+	jsonTable.push([key, darwin[key], win32[key]]);
+});
+
+const figureTable = table(jsonTable, {align: ['', 'c', 'c']});
+
+let readme = fs.readFileSync('readme.md', 'utf8');
+readme = readme.replace(/## Figures[^#]*/gm, `## Figures\n\n${figureTable}\n\n\n`);
```
Comment (SamVerschueren) [inline makefile.js]: The regex might not be 100% future proof though. For instance if a figure is added that has a fallback with a `#`. But I guess it's good enough for now.

Comment (sindresorhus) [inline makefile.js]: More than good enough ;)

**Comment by sindresorhus** [comment]
> One hour later and it's still running :D.

One of many reasons I don't add AppVeyor to every repo.

**Comment by sindresorhus** [comment]
5fa79994-b5ee-11e3-9a4d-b19557d7480b

**Comment by SamVerschueren** [comment]
Hahaha awesome! :)

**Pull request merged**